frontend/js/main.js

```diff
@@ -0,0 +1,2203 @@
+
+    ddlCardCollapse.appendChild(ddlCardContent)
+    ddlCard.appendChild(ddlCardCollapse)
+
+    li.appendChild(ddlCard);
+    ddlUl.appendChild(li)
+  }
+  ddl_accordion.appendChild(ddlUl);
+}
+
+/**
+ * Function to render edit schema screen from connect to DB mode
+ *
+ * @param {event} windowEvent hashchange or load event
+ * @return {null}
+ */
+async function showSchemaAssessment(windowEvent) {
+  showSpinner();
+  reportData = await fetch(apiUrl + '/convert/infoschema')
+  .then(function (response) {
+    if (response.ok) {
+      return response;
+    }
+    else {
+      return Promise.reject(response);
+    }
+  })
+  .catch(function (err) {
+    showSnackbar(err, ' redBg');
+  });
+  reportDataResp = await reportData.json();
+  localStorage.setItem('conversionReportContent', JSON.stringify(reportDataResp));
+  await ddlSummaryAndConversionApiCall();
+  await getInterleaveInfo();
+  sourceTableFlag = localStorage.getItem('sourceDbName');
+  jQuery('#connectModalSuccess').modal("hide");
+  jQuery('#connectToDbModal').modal("hide");
+  jQuery('#globalDataTypeModal').modal("hide");
+  const { component = ErrorComponent } = findComponentByPath('/schema-report-connect-to-db', routes) || {};
+  document.getElementById('app').innerHTML = component.render();
+  showSchemaConversionReportContent(windowEvent);
+  sessionRetrieval(sourceTableFlag);
+  showSnackbar('schema converted successfully !!', ' greenBg');
+}
+
+/**
+ * Function to make ddl and summary api calls
+ *
+ * @return {null}
+ */
+async function ddlSummaryAndConversionApiCall() {
+  ddlData = await fetch(apiUrl + '/ddl')
+  .then(function (response) {
+    if (response.ok) {
+      return response;
+    }
+    else {
+      return Promise.reject(response);
+    }
+  })
+  .catch(function (err) {
+    showSnackbar(err, ' redBg');
+  });
+  ddlDataResp = await ddlData.json();
+  localStorage.setItem('ddlStatementsContent', JSON.stringify(ddlDataResp));
+
+  summaryData = await fetch(apiUrl + '/summary')
+  .then(function (response) {
+    if (response.ok) {
+      return response;
+    }
+    else {
+      return Promise.reject(response);
+    }
+  })
+  .catch(function (err) {
+    showSnackbar(err, ' redBg');
+  });
+  summaryDataResp = await summaryData.json();
+  localStorage.setItem('summaryReportContent', JSON.stringify(summaryDataResp));
+
+  conversionRate = await fetch(apiUrl + '/conversion')
+  .then(function (response) {
+    if (response.ok) {
+      return response;
+    }
+    else {
+      return Promise.reject(response);
+    }
+  })
+  .catch(function (err) {
+    showSnackbar(err, ' redBg');
+  });
+  conversionRateResp = await conversionRate.json();
+  localStorage.setItem('tableBorderColor', JSON.stringify(conversionRateResp));
+}
+
+/**
+ * Function to call create tables function for edit schema screen
+ *
+ * @param {event} windowEvent hashchange or load event
+ * @return {null}
+ */
+const showSchemaConversionReportContent = (windowEvent) => {
```

Comment:
windowEvent is not used here. we can safely remove that and also from where this function is called.

Reply:
windowEvent is removed